lib/terraforming/resource/route_table.rb

```diff
@@ -52,6 +59,73 @@ def module_name_of(route_table)
       def route_tables
         @client.describe_route_tables.route_tables
       end
+
+      def routes_attributes_of(route_table)
+        routes = routes_of(route_table)
+        attributes = { "route.#" => routes.length.to_s }
+
+        routes.each do |route|
+          attributes.merge!(route_attributes_of(route))
+        end
+
+        attributes
+      end
+
+      def route_attributes_of(route)
+        hashcode = route_hashcode_of(route)
+        attributes = {
+          "route.#{hashcode}.cidr_block" => route.destination_cidr_block.to_s,
+          "route.#{hashcode}.gateway_id" => route.gateway_id.to_s,
+          "route.#{hashcode}.instance_id" => route.instance_id.to_s,
+          "route.#{hashcode}.network_interface_id" => route.network_interface_id.to_s,
+          "route.#{hashcode}.vpc_peering_connection_id" => route.vpc_peering_connection_id.to_s
+        }
+
+        attributes
+      end
+
+      def route_hashcode_of(route)
+        string = "#{route.destination_cidr_block}-" <<
+                 "#{route.gateway_id}-"
+
+        instance_set = false
+        if route.instance_id != ''
+          string += route.instance_id.to_s
```

Comment:
Please use `<<` instead of `+=` to append string. `<<` is more faster than `+=`.

## REF
- [coderwall.com : establishing geek cred since 1305712800](https://coderwall.com/p/ac5j9g/or-concat-what-is-faster-for-appending-string-in-ruby)